src/LPOracle.sol

```diff
@@ -60,6 +60,66 @@ contract LPOracle {
         TOKEN1_DECIMALS = TOKEN1.decimals();
     }
 
+    /*----------------------------------------------------------*|
+    |*  # AGGREGATOR V3 INTERFACE REQUIREMENTS                  *|
+    |*----------------------------------------------------------*/
+
+    /// @notice Returns the number of decimals used.
+    function decimals() external pure returns (uint8) {
+        return 8;
+    }
+
+    /// @notice Returns the description of the LP token pricing oracle.
+    function description() external view returns (string memory) {
+        return string.concat(IERC20(POOL).name(), " LP Token / USD");
+    }
+
+    /// @notice Returns the oracle version.
+    /// @dev Chainlink interface requires implementation.
+    function version() external pure returns (uint256) {
+        return 0;
+    }
+
+    /// @dev Chainlink interface requires implementation. No meaningful values for this contract.
+    function getRoundData(uint80)
+        external
+        pure
+        returns (uint80 roundId, int256 answer, uint256 startedAt, uint256 updatedAt, uint80 answeredInRound)
+    {
+        return (0, 0, 0, 0, 0);
```

Comment:
Should we use some real values here? I would imagine something like block number for round id and the other return parameters seem relevant. 

I think we should log an issue for the implementation of this.

Reply:
Definitely add an issue for this.

[Here](https://docs.chain.link/chainlink-local/api-reference/solidity/data-feeds/interfaces/aggregatorv3interface) is the docs for the AggregatorV3Interface. This function is used for getting the data for a specific roundId. Now, our oracle relies on two price feeds with presumably separate roundIds.

I was unsure how, and if we would even need to, implement this function. 

Lending protocols, for example, would be calling the latestRoundData function.